graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/fetchers/admin/SingleSchemaFetcher.java

```diff
@@ -46,8 +46,8 @@ protected SchemaSource get(DataFetchingEnvironment environment, StargateGraphqlC
 
     authorize(context, keyspace);
 
-    return schemaSourceDaoProvider
-        .apply(context.getDataStore())
-        .getSingleVersion(keyspace, version);
+    SchemaSource schemaSource =
+        schemaSourceDaoProvider.apply(context.getDataStore()).getSingleVersion(keyspace, version);
+    return schemaSource;
```

Comment:
Was this change to debug that `null` was being returned?

Reply:
It's a remnant from an earlier attempt. I'll revert it.